### ISSUE TITLE
The like buttons aren't appearing on the website for me.

### ISSUE THREAD

**Issue by StinkerB06** [nesbox/TIC-80#758]
This has happened on Google Chrome on the computers I have in my possession.
![bandicam 2018-10-14 06-13-50-632](https://user-images.githubusercontent.com/30059831/46917228-4cb93a00-cf79-11e8-9ace-a0d377742f2d.jpg)
This applies to pretty much every cart I've seen on the site so far since it began to happen.

**Comment by ElfEars** [comment]
Same for me on firefox

**Comment by StinkerB06** [comment]
OK, I just found out why the button was blocked. One of my installed ad blockers, NoAds, blocks the button every time I have it running on the website.
If you want to like a cartridge on the site, click on the NoAds extension's icon (should look like a green square with a shield), click "Don't run on pages on this domain", then it asks for what specific pages of the domain to not have the ad blocker run on. Move the slider to select **_tic.computer/*_** or **_tic.computer/play_**, then click **Exclude**. Reload the page. The button should now be there, unless other ad blockers are blocking it.

Closing...
